web/src/echo.js

```diff
@@ -84,8 +87,11 @@ function resend_message(message, $row) {
         const locally_echoed = true;
 
         hide_retry_spinner($row);
+        function send_message_success(local_id, message_id, locally_echoed) {
+            return send_message_success(local_id, message_id, locally_echoed);
+        }
```

Comment:
This can't possibly be right.

Reply:
fixed it

Reply:
> @timabbott Im having issues with tests after refactoring this code and moving `send_message_success` from compose to echo and just passing a callback, any way on how you can guide me to fix it?

and can I import the function from `echo.js` in the `compose.js` tests?